native/CMakeLists.txt

```diff
@@ -2459,8 +2468,8 @@ if(USE_MIDDLEWARE)
                                      cocos/editor-support/spine-creator-support/SkeletonCacheMgr.h
             NO_WERROR                cocos/editor-support/spine-creator-support/SkeletonDataMgr.cpp
                                      cocos/editor-support/spine-creator-support/SkeletonDataMgr.h
-            NO_WERROR   NO_UBUILD    cocos/editor-support/spine-creator-support/SkeletonRenderer.cpp
-                                     cocos/editor-support/spine-creator-support/SkeletonRenderer.h
+            NO_WERROR   NO_UBUILD    cocos/editor-support/spine-creator-support/CCSkeletonRenderer.cpp
+                                     cocos/editor-support/spine-creator-support/CCSkeletonRenderer.h
```

Comment:
Why add `CC` prefix for the file name?

Reply:
SkeletonRender has been exist after version 4.2